/cm/shared/apps/python/3.5.2/lib/python3.5/site-packages/h5py/__init__.py:34: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
  from ._conv import register_converters as _register_converters
parameters: {'shuffle': True, 'depth': 9, 'model_load_path': 'models/VDCNN/ag_news-2000_model.pt', 'model_save_path': 'models/VDCNN/VDCNN_ng20---ag_news_depth@9/1', 'joint_training': True, 'lr': 0.01, 'transfer_lr': 0.001, 'chunk_size': 2048, 'model_folder': 'models/VDCNN/VDCNN_ng20---ag_news_depth@9/1', 'test_interval': 1000, 'num_embedding_features': 100, 'joint_test': 0, 'last_pooling_layer': 'k-max-pooling', 'test_dataset': 'ng20', 'gpu': True, 'maxlen': 1024, 'freeze_pre_trained_layers': False, 'num_prev_classes': 20, 'shortcut': False, 'batch_size': 128, 'lr_halve_interval': 15000.0, 'combined_datasets': 'ng20---ag_news', 'class_weights': None, 'test_only': 0, 'transfer_weights': False, 'test_batch_size': 128, 'dataset': 'ag_news', 'seed': 1337, 'validation_ratio': 0.1, 'iterations': 50000, 'joint_ratio': 0.01}
Joint Training !
dataset: Newsgroup20, n_classes: 20
  - loading dataset...
dataset: AgNews, n_classes: 4
  - loading dataset...
Both datasets loaded, going to mix ...
Transfer Data Size: 120000
Transfer Ratio Size: 1200
Root dataset Test length - sentences 5645, labels 5645
Transfer dataset Test length - sentences 7600, labels 7600
Unused dataset Test length - sentences 118800, labels 118800
Mixed dataset Test length - sentences 132045, labels 132045
Mixed dataset Train length - sentences 14379, labels 14379
 Joint Training: Txt vectorization...
--- Logging error ---
Traceback (most recent call last):
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 980, in emit
    msg = self.format(record)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 830, in format
    return fmt.format(record)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 567, in format
    record.message = record.getMessage()
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 330, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Call stack:
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/runpy.py", line 184, in _run_module_as_main
    "__main__", mod_spec)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/usaxena/work/s18/lex/code/vdcnn/src/main.py", line 87, in <module>
    joint_train(opt, logger)
  File "/home/usaxena/work/s18/lex/code/vdcnn/src/VDCNN.py", line 282, in joint_train
    logger.info("n_txt_feats before overriding: ", n_txt_feats)
Message: 'n_txt_feats before overriding: '
Arguments: (87,)
--- Logging error ---
Traceback (most recent call last):
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 980, in emit
    msg = self.format(record)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 830, in format
    return fmt.format(record)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 567, in format
    record.message = record.getMessage()
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 330, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Call stack:
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/runpy.py", line 184, in _run_module_as_main
    "__main__", mod_spec)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/usaxena/work/s18/lex/code/vdcnn/src/main.py", line 87, in <module>
    joint_train(opt, logger)
  File "/home/usaxena/work/s18/lex/code/vdcnn/src/VDCNN.py", line 282, in joint_train
    logger.info("n_txt_feats before overriding: ", n_txt_feats)
Message: 'n_txt_feats before overriding: '
Arguments: (87,)
--- Logging error ---
Traceback (most recent call last):
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 980, in emit
    msg = self.format(record)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 830, in format
    return fmt.format(record)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 567, in format
    record.message = record.getMessage()
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 330, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Call stack:
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/runpy.py", line 184, in _run_module_as_main
    "__main__", mod_spec)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/usaxena/work/s18/lex/code/vdcnn/src/main.py", line 87, in <module>
    joint_train(opt, logger)
  File "/home/usaxena/work/s18/lex/code/vdcnn/src/VDCNN.py", line 284, in joint_train
    logger.info("Number of classes in the mixed dataset are: ", n_classes, type(n_classes))
Message: 'Number of classes in the mixed dataset are: '
Arguments: (24, <class 'int'>)
--- Logging error ---
Traceback (most recent call last):
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 980, in emit
    msg = self.format(record)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 830, in format
    return fmt.format(record)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 567, in format
    record.message = record.getMessage()
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 330, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Call stack:
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/runpy.py", line 184, in _run_module_as_main
    "__main__", mod_spec)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/usaxena/work/s18/lex/code/vdcnn/src/main.py", line 87, in <module>
    joint_train(opt, logger)
  File "/home/usaxena/work/s18/lex/code/vdcnn/src/VDCNN.py", line 284, in joint_train
    logger.info("Number of classes in the mixed dataset are: ", n_classes, type(n_classes))
Message: 'Number of classes in the mixed dataset are: '
Arguments: (24, <class 'int'>)
--- Logging error ---
Traceback (most recent call last):
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 980, in emit
    msg = self.format(record)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 830, in format
    return fmt.format(record)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 567, in format
    record.message = record.getMessage()
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 330, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Call stack:
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/runpy.py", line 184, in _run_module_as_main
    "__main__", mod_spec)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/usaxena/work/s18/lex/code/vdcnn/src/main.py", line 87, in <module>
    joint_train(opt, logger)
  File "/home/usaxena/work/s18/lex/code/vdcnn/src/VDCNN.py", line 287, in joint_train
    logger.info("Seed for random numbers: ", torch.initial_seed())
Message: 'Seed for random numbers: '
Arguments: (1337,)
--- Logging error ---
Traceback (most recent call last):
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 980, in emit
    msg = self.format(record)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 830, in format
    return fmt.format(record)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 567, in format
    record.message = record.getMessage()
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 330, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Call stack:
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/runpy.py", line 184, in _run_module_as_main
    "__main__", mod_spec)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/usaxena/work/s18/lex/code/vdcnn/src/main.py", line 87, in <module>
    joint_train(opt, logger)
  File "/home/usaxena/work/s18/lex/code/vdcnn/src/VDCNN.py", line 287, in joint_train
    logger.info("Seed for random numbers: ", torch.initial_seed())
Message: 'Seed for random numbers: '
Arguments: (1337,)
--- Logging error ---
Traceback (most recent call last):
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 980, in emit
    msg = self.format(record)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 830, in format
    return fmt.format(record)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 567, in format
    record.message = record.getMessage()
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 330, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Call stack:
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/runpy.py", line 184, in _run_module_as_main
    "__main__", mod_spec)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/usaxena/work/s18/lex/code/vdcnn/src/main.py", line 87, in <module>
    joint_train(opt, logger)
  File "/home/usaxena/work/s18/lex/code/vdcnn/src/VDCNN.py", line 290, in joint_train
    logger.info("Overriding the number of embedding features to: ", n_txt_feats)
Message: 'Overriding the number of embedding features to: '
Arguments: (100,)
--- Logging error ---
Traceback (most recent call last):
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 980, in emit
    msg = self.format(record)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 830, in format
    return fmt.format(record)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 567, in format
    record.message = record.getMessage()
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 330, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Call stack:
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/runpy.py", line 184, in _run_module_as_main
    "__main__", mod_spec)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/usaxena/work/s18/lex/code/vdcnn/src/main.py", line 87, in <module>
    joint_train(opt, logger)
  File "/home/usaxena/work/s18/lex/code/vdcnn/src/VDCNN.py", line 290, in joint_train
    logger.info("Overriding the number of embedding features to: ", n_txt_feats)
Message: 'Overriding the number of embedding features to: '
Arguments: (100,)
Joint training
Setting the lr to : 0.01
/home/usaxena/work/s18/lex/code/vdcnn/src/VDCNN.py:148: UserWarning: Implicit dimension choice for softmax has been deprecated. Change the call to include dim=X as an argument.
  yhat = nn.functional.softmax(yhat)
Mixed_ng20---ag_news_0.01 - Iter [0/50000]: train-metrics- {'accuracy': 0.0625, 'logloss': -1} ; val-metrics- {'accuracy': 0.05006954102920723, 'logloss': -1}
Mixed_ng20---ag_news_0.01 - Iter [1000/50000]: train-metrics- {'accuracy': 0.2265625, 'logloss': -1} ; val-metrics- {'accuracy': 0.1933240611961057, 'logloss': -1}
Mixed_ng20---ag_news_0.01 - Iter [2000/50000]: train-metrics- {'accuracy': 0.6171875, 'logloss': -1} ; val-metrics- {'accuracy': 0.3428372739916551, 'logloss': -1}
Mixed_ng20---ag_news_0.01 - Iter [3000/50000]: train-metrics- {'accuracy': 0.8046875, 'logloss': -1} ; val-metrics- {'accuracy': 0.541029207232267, 'logloss': -1}
Mixed_ng20---ag_news_0.01 - Iter [4000/50000]: train-metrics- {'accuracy': 0.8359375, 'logloss': -1} ; val-metrics- {'accuracy': 0.28303198887343534, 'logloss': -1}
Mixed_ng20---ag_news_0.01 - Iter [5000/50000]: train-metrics- {'accuracy': 0.9296875, 'logloss': -1} ; val-metrics- {'accuracy': 0.1905424200278164, 'logloss': -1}
Mixed_ng20---ag_news_0.01 - Iter [6000/50000]: train-metrics- {'accuracy': 0.9609375, 'logloss': -1} ; val-metrics- {'accuracy': 0.6703755215577191, 'logloss': -1}
Mixed_ng20---ag_news_0.01 - Iter [7000/50000]: train-metrics- {'accuracy': 1.0, 'logloss': -1} ; val-metrics- {'accuracy': 0.6905424200278164, 'logloss': -1}
Mixed_ng20---ag_news_0.01 - Iter [8000/50000]: train-metrics- {'accuracy': 0.984375, 'logloss': -1} ; val-metrics- {'accuracy': 0.694019471488178, 'logloss': -1}
Mixed_ng20---ag_news_0.01 - Iter [9000/50000]: train-metrics- {'accuracy': 1.0, 'logloss': -1} ; val-metrics- {'accuracy': 0.7079276773296245, 'logloss': -1}
Mixed_ng20---ag_news_0.01 - Iter [10000/50000]: train-metrics- {'accuracy': 0.953125, 'logloss': -1} ; val-metrics- {'accuracy': 0.3463143254520167, 'logloss': -1}
Mixed_ng20---ag_news_0.01 - Iter [11000/50000]: train-metrics- {'accuracy': 1.0, 'logloss': -1} ; val-metrics- {'accuracy': 0.7635605006954103, 'logloss': -1}
Mixed_ng20---ag_news_0.01 - Iter [12000/50000]: train-metrics- {'accuracy': 1.0, 'logloss': -1} ; val-metrics- {'accuracy': 0.7698191933240612, 'logloss': -1}
Mixed_ng20---ag_news_0.01 - Iter [13000/50000]: train-metrics- {'accuracy': 1.0, 'logloss': -1} ; val-metrics- {'accuracy': 0.7684283727399166, 'logloss': -1}
Mixed_ng20---ag_news_0.01 - Iter [14000/50000]: train-metrics- {'accuracy': 1.0, 'logloss': -1} ; val-metrics- {'accuracy': 0.7698191933240612, 'logloss': -1}
Mixed_ng20---ag_news_0.01 - Iter [15000/50000]: train-metrics- {'accuracy': 1.0, 'logloss': -1} ; val-metrics- {'accuracy': 0.6161335187760779, 'logloss': -1}
new lr: 0.005
Mixed_ng20---ag_news_0.01 - Iter [16000/50000]: train-metrics- {'accuracy': 1.0, 'logloss': -1} ; val-metrics- {'accuracy': 0.7712100139082059, 'logloss': -1}
Mixed_ng20---ag_news_0.01 - Iter [17000/50000]: train-metrics- {'accuracy': 1.0, 'logloss': -1} ; val-metrics- {'accuracy': 0.7677329624478443, 'logloss': -1}
Mixed_ng20---ag_news_0.01 - Iter [18000/50000]: train-metrics- {'accuracy': 1.0, 'logloss': -1} ; val-metrics- {'accuracy': 0.7712100139082059, 'logloss': -1}
Mixed_ng20---ag_news_0.01 - Iter [19000/50000]: train-metrics- {'accuracy': 1.0, 'logloss': -1} ; val-metrics- {'accuracy': 0.7691237830319889, 'logloss': -1}
Mixed_ng20---ag_news_0.01 - Iter [20000/50000]: train-metrics- {'accuracy': 1.0, 'logloss': -1} ; val-metrics- {'accuracy': 0.6801112656467315, 'logloss': -1}
Mixed_ng20---ag_news_0.01 - Iter [21000/50000]: train-metrics- {'accuracy': 1.0, 'logloss': -1} ; val-metrics- {'accuracy': 0.7712100139082059, 'logloss': -1}
Mixed_ng20---ag_news_0.01 - Iter [22000/50000]: train-metrics- {'accuracy': 1.0, 'logloss': -1} ; val-metrics- {'accuracy': 0.7698191933240612, 'logloss': -1}
Mixed_ng20---ag_news_0.01 - Iter [23000/50000]: train-metrics- {'accuracy': 1.0, 'logloss': -1} ; val-metrics- {'accuracy': 0.7698191933240612, 'logloss': -1}
Mixed_ng20---ag_news_0.01 - Iter [24000/50000]: train-metrics- {'accuracy': 1.0, 'logloss': -1} ; val-metrics- {'accuracy': 0.7705146036161336, 'logloss': -1}
Mixed_ng20---ag_news_0.01 - Iter [25000/50000]: train-metrics- {'accuracy': 1.0, 'logloss': -1} ; val-metrics- {'accuracy': 0.7051460361613352, 'logloss': -1}
Mixed_ng20---ag_news_0.01 - Iter [26000/50000]: train-metrics- {'accuracy': 1.0, 'logloss': -1} ; val-metrics- {'accuracy': 0.7566063977746871, 'logloss': -1}
Mixed_ng20---ag_news_0.01 - Iter [27000/50000]: train-metrics- {'accuracy': 1.0, 'logloss': -1} ; val-metrics- {'accuracy': 0.7726008344923505, 'logloss': -1}
Mixed_ng20---ag_news_0.01 - Iter [28000/50000]: train-metrics- {'accuracy': 1.0, 'logloss': -1} ; val-metrics- {'accuracy': 0.7705146036161336, 'logloss': -1}
Mixed_ng20---ag_news_0.01 - Iter [29000/50000]: train-metrics- {'accuracy': 1.0, 'logloss': -1} ; val-metrics- {'accuracy': 0.7691237830319889, 'logloss': -1}
Mixed_ng20---ag_news_0.01 - Iter [30000/50000]: train-metrics- {'accuracy': 1.0, 'logloss': -1} ; val-metrics- {'accuracy': 0.7329624478442281, 'logloss': -1}
new lr: 0.0025
Mixed_ng20---ag_news_0.01 - Iter [31000/50000]: train-metrics- {'accuracy': 1.0, 'logloss': -1} ; val-metrics- {'accuracy': 0.6397774687065368, 'logloss': -1}
Mixed_ng20---ag_news_0.01 - Iter [32000/50000]: train-metrics- {'accuracy': 1.0, 'logloss': -1} ; val-metrics- {'accuracy': 0.7719054242002782, 'logloss': -1}
Mixed_ng20---ag_news_0.01 - Iter [33000/50000]: train-metrics- {'accuracy': 1.0, 'logloss': -1} ; val-metrics- {'accuracy': 0.7712100139082059, 'logloss': -1}
Mixed_ng20---ag_news_0.01 - Iter [34000/50000]: train-metrics- {'accuracy': 1.0, 'logloss': -1} ; val-metrics- {'accuracy': 0.7726008344923505, 'logloss': -1}
Mixed_ng20---ag_news_0.01 - Iter [35000/50000]: train-metrics- {'accuracy': 1.0, 'logloss': -1} ; val-metrics- {'accuracy': 0.7454798331015299, 'logloss': -1}
Mixed_ng20---ag_news_0.01 - Iter [36000/50000]: train-metrics- {'accuracy': 1.0, 'logloss': -1} ; val-metrics- {'accuracy': 0.4993045897079277, 'logloss': -1}
Mixed_ng20---ag_news_0.01 - Iter [37000/50000]: train-metrics- {'accuracy': 1.0, 'logloss': -1} ; val-metrics- {'accuracy': 0.7719054242002782, 'logloss': -1}
Mixed_ng20---ag_news_0.01 - Iter [38000/50000]: train-metrics- {'accuracy': 1.0, 'logloss': -1} ; val-metrics- {'accuracy': 0.7712100139082059, 'logloss': -1}
Mixed_ng20---ag_news_0.01 - Iter [39000/50000]: train-metrics- {'accuracy': 1.0, 'logloss': -1} ; val-metrics- {'accuracy': 0.7719054242002782, 'logloss': -1}
Mixed_ng20---ag_news_0.01 - Iter [40000/50000]: train-metrics- {'accuracy': 1.0, 'logloss': -1} ; val-metrics- {'accuracy': 0.7517385257301809, 'logloss': -1}
Mixed_ng20---ag_news_0.01 - Iter [41000/50000]: train-metrics- {'accuracy': 1.0, 'logloss': -1} ; val-metrics- {'accuracy': 0.3623087621696801, 'logloss': -1}
Mixed_ng20---ag_news_0.01 - Iter [42000/50000]: train-metrics- {'accuracy': 1.0, 'logloss': -1} ; val-metrics- {'accuracy': 0.7712100139082059, 'logloss': -1}
Mixed_ng20---ag_news_0.01 - Iter [43000/50000]: train-metrics- {'accuracy': 1.0, 'logloss': -1} ; val-metrics- {'accuracy': 0.7712100139082059, 'logloss': -1}
Mixed_ng20---ag_news_0.01 - Iter [44000/50000]: train-metrics- {'accuracy': 1.0, 'logloss': -1} ; val-metrics- {'accuracy': 0.7712100139082059, 'logloss': -1}
Mixed_ng20---ag_news_0.01 - Iter [45000/50000]: train-metrics- {'accuracy': 1.0, 'logloss': -1} ; val-metrics- {'accuracy': 0.7579972183588317, 'logloss': -1}
new lr: 0.00125
Mixed_ng20---ag_news_0.01 - Iter [46000/50000]: train-metrics- {'accuracy': 1.0, 'logloss': -1} ; val-metrics- {'accuracy': 0.27885952712100137, 'logloss': -1}
Mixed_ng20---ag_news_0.01 - Iter [47000/50000]: train-metrics- {'accuracy': 1.0, 'logloss': -1} ; val-metrics- {'accuracy': 0.7712100139082059, 'logloss': -1}
Mixed_ng20---ag_news_0.01 - Iter [48000/50000]: train-metrics- {'accuracy': 1.0, 'logloss': -1} ; val-metrics- {'accuracy': 0.7712100139082059, 'logloss': -1}
Mixed_ng20---ag_news_0.01 - Iter [49000/50000]: train-metrics- {'accuracy': 1.0, 'logloss': -1} ; val-metrics- {'accuracy': 0.7719054242002782, 'logloss': -1}
After Training: Testing on root dataset only
Traceback (most recent call last):
  File "/home/usaxena/.local/lib/python3.5/site-packages/torch/nn/modules/module.py", line 482, in load_state_dict
    own_state[name].copy_(param)
RuntimeError: invalid argument 2: sizes do not match at /pytorch/torch/lib/THC/generic/THCTensorCopy.c:51

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/runpy.py", line 184, in _run_module_as_main
    "__main__", mod_spec)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/usaxena/work/s18/lex/code/vdcnn/src/main.py", line 87, in <module>
    joint_train(opt, logger)
  File "/home/usaxena/work/s18/lex/code/vdcnn/src/VDCNN.py", line 314, in joint_train
    test(model, logger, opt, root_te_data, n_classes, dataset_name)
  File "/home/usaxena/work/s18/lex/code/vdcnn/src/VDCNN.py", line 261, in test
    model.load_state_dict(checkpoint['model'])
  File "/home/usaxena/.local/lib/python3.5/site-packages/torch/nn/modules/module.py", line 487, in load_state_dict
    .format(name, own_state[name].size(), param.size()))
RuntimeError: While copying the parameter named embed.weight, whose dimensions in the model are torch.Size([100, 16]) and whose dimensions in the checkpoint are torch.Size([66, 16]).
